### PR TITLE
Update theme documentation about the considerations when deploying custom themes

### PR DESCRIPTION
**Pull request by pedroigor**
Related #23907

### CODE REVIEW

**Review by stianst — APPROVED** [review]

File: docs/documentation/server_development/topics/themes.adoc [hunk 1]
```diff
@@ -345,8 +346,18 @@ Icons for social providers are already defined in `base` login theme properties
...
+Although it is possible to create custom templates to change completely how pages are rendered, the recommendation is to leverage the built-in
+templates as much as possible. The reasons are:
+
+* During upgrades, you might be forced to update your custom templates to get the latest updates from newer versions
+* link:#_theme_stylesheet[Configuring CSS styles] to your themes allows you to adapt the UI to match your UI design standards and guidelines.
+* link:{adminguide_link}#user-profile[User Profile] allows you to support custom user attributes and configure how they are rendered.
+
+In most cases, you won't need to change templates to adapt {project_name} to your needs, but you can override individual
+templates in your own theme by creating `<THEME TYPE>/<TEMPLATE>.ftl`.
+Admin and account console use a single template `index.ftl` for rendering the application. For a list of templates in other theme types used see `themes/base/<THEME TYPE>`.
```
Comment (stianst) [inline docs/documentation/server_development/topics/themes.adoc:360]: The templates are not available in `themes/base`, but rather in `lib/lib/main/org.keycloak.keycloak-themes-<VERSION>.jar`

**Review by stianst — CHANGES_REQUESTED** [review]
Need to change instructions on how to list templates

File: docs/documentation/server_development/topics/themes.adoc (hunk 1 above)
Comment (stianst) [reply docs/documentation/server_development/topics/themes.adoc:360]: Perhaps link to the default themes section? Maybe we could also update that to mention where this jar is?

Comment (pedroigor) [reply docs/documentation/server_development/topics/themes.adoc]: Done

**Review by stianst — APPROVED** [review]

**Pull request merged**